pages/api/allocated-cases.js

```diff
@@ -0,0 +1,33 @@
+import * as HttpStatus from 'http-status-codes';
+
+import { isAuthorised } from 'utils/auth';
+import { cases } from 'fixture/allocatedTestData';
```

Comment:
Instead of importing the fixture here it would be nice having the BE API integration (`/utils/server/), so we don't need to add it later when the BE is ready.